scripts/ChangesSinceLastRelease.txt

```diff
@@ -368,3 +368,79 @@ Windows.Win32.UI.Magnification.Apis.MW_FILTERMODE_INCLUDE removed
+Windows.Win32.Graphics.Gdi.Apis.OUT_SCREEN_OUTLINE_PRECIS removed
+Windows.Win32.Graphics.Gdi.Apis.VARIABLE_PITCH removed
+Windows.Win32.Graphics.Gdi.FONT_CHARSET.value__...System.UInt32 => System.Byte
+Windows.Win32.Graphics.Gdi.FONT_CLIP_PRECISION.CLIP_DFA_OVERRIDE added
+Windows.Win32.Graphics.Gdi.FONT_CLIP_PRECISION.value__...System.UInt32 => System.Byte
+Windows.Win32.Graphics.Gdi.FONT_FAMILY added
+Windows.Win32.Graphics.Gdi.FONT_FAMILY.FF_DECORATIVE added
+Windows.Win32.Graphics.Gdi.FONT_FAMILY.FF_DONTCARE added
+Windows.Win32.Graphics.Gdi.FONT_FAMILY.FF_MODERN added
+Windows.Win32.Graphics.Gdi.FONT_FAMILY.FF_ROMAN added
+Windows.Win32.Graphics.Gdi.FONT_FAMILY.FF_SCRIPT added
+Windows.Win32.Graphics.Gdi.FONT_FAMILY.FF_SWISS added
+Windows.Win32.Graphics.Gdi.FONT_OUTPUT_PRECISION.OUT_SCREEN_OUTLINE_PRECIS added
+Windows.Win32.Graphics.Gdi.FONT_OUTPUT_PRECISION.value__...System.UInt32 => System.Byte
+Windows.Win32.Graphics.Gdi.FONT_PITCH added
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY.FF_DECORATIVE removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY.FF_DONTCARE removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY.FF_MODERN removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY.FF_ROMAN removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY.FF_SCRIPT removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY.FF_SWISS removed
+Windows.Win32.Graphics.Gdi.FONT_PITCH.DEFAULT_PITCH added
+Windows.Win32.Graphics.Gdi.FONT_PITCH.FIXED_PITCH added
+Windows.Win32.Graphics.Gdi.FONT_PITCH.VARIABLE_PITCH added
+Windows.Win32.Graphics.Gdi.FONT_QUALITY.value__...System.UInt32 => System.Byte
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_BLACK removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_BOLD removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_DEMIBOLD removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_DONTCARE removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_EXTRABOLD removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_EXTRALIGHT removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_HEAVY removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_LIGHT removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_MEDIUM removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_NORMAL removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_REGULAR removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_SEMIBOLD removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_THIN removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_ULTRABOLD removed
+Windows.Win32.Graphics.Gdi.FONT_WEIGHT.FW_ULTRALIGHT removed
+Windows.Win32.Graphics.Gdi.LOGFONTA.lfCharSet...System.Byte => Windows.Win32.Graphics.Gdi.FONT_CHARSET
+Windows.Win32.Graphics.Gdi.LOGFONTA.lfPitchAndFamily...Windows.Win32.Graphics.Gdi.FONT_PITCH_AND_FAMILY => System.Byte
+Windows.Win32.Graphics.Gdi.LOGFONTA.lfWeight...Windows.Win32.Graphics.Gdi.FONT_WEIGHT => System.Int32
```

Comment:
Why did we remove the enum from this field? The enum was already 4 bytes long.

Reply:
From docs:
> **The weight of the font in the range 0 through 1000.** For example, 400 is normal and 700 is bold. If this value is zero, a default weight is used.
> 
> **The following values are defined for convenience.**

https://learn.microsoft.com/en-us/windows/win32/api/wingdi/ns-wingdi-logfonta